src/core/reducers/infoDialog.js

```diff
@@ -1,16 +1,58 @@
-import { CLOSE_INFO, SHOW_INFO } from 'core/constants';
+/* @flow */
+export const CLOSE_INFO: 'CLOSE_INFO' = 'CLOSE_INFO';
+export const SHOW_INFO: 'SHOW_INFO' = 'SHOW_INFO';
 
-export default function infoDialog(state = {}, { type, payload }) {
-  switch (type) {
+type InfoDialogData = {|
+  addonName: string,
+  closeAction: Function,
```

Comment:
Can you file an issue to stop dispatching a function in the action like this? It is very bad practice because it breaks any feature that relies on serializing the action stack.

If you have an idea for a lint rule to prevent anyone from dispatching a function, that would be nice.

Reply:
https://redux.js.org/basics/actions

> Actions are plain JavaScript objects

Reply:
Yup, I have it on my TODO post-it. Thanks for the review!

Reply:
Done!